packages/editor-ui/src/composables/useCanvasOperations.ts

```diff
@@ -365,7 +364,7 @@ export function useCanvasOperations({ router }: { router: ReturnType<typeof useR
 		if (node.type === STICKY_NODE_TYPE) {
 			telemetry.track('User deleted workflow note', {
 				workflow_id: workflowsStore.workflowId,
-				is_welcome_note: node.name === QUICKSTART_NOTE_NAME,
+				is_welcome_note: false,
```

Comment:
do we still need 'is_welcome_note'?

Reply:
I did not want to change it for telemetry to be consistent, but not really anymore.